cves/CVE-2010-1157.yml

```diff
@@ -99,9 +105,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: false
```

Comment:
I think for the "answer" field you are supposed to give a description as to what the state of unit testing was at the time of the fix.

Answer field from Meneely's example: "From looking at the fix code it's clear that code was tested and they added a new test for fixing this vulnerability."

Reply:
Thanks, I missed that one.